v2/tools/generator/internal/functions/testdata/TestGolden_GetExtendedResourceFunction_moreThanOneVersion_GeneratesExpectedCode/GetExtendedResources.golden

```diff
@@ -1,7 +1,7 @@
 // Code generated by azure-service-operator-codegen. DO NOT EDIT.
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-package customizations
+package vcustomizations
```

Comment:
That doesn't look ideal - customizations should not get `v` prefixed.

Reply:
Fixed.